README.md

```diff
@@ -26,11 +26,14 @@ A curated list of awesome things and projects built by Algerian developpers.
   - [Phyber](https://github.com/ghousseyn/phiber) - Build light-weight lightning-fast PhP applications with Phiber
 - Go
   - [Go Arabic](https://github.com/01walid/goarabic) - A Go Lang package for dealing with Arabic text
+- C
+  - [ArduinoSpritzCipher](https://github.com/abderraouf-adjal/ArduinoSpritzCipher) - Spritz library for Arduino, CSPRNG, cryptographic hash and MAC functions, symmetric-key data encryption, and general-purpose functions
```

Comment:
I'm not sure I understand really why you can do any change to the project, I was asking for just a LICENSE file (and not touching the code :) ). But whatever, this is not blocking since you mentioned the license in your README file. I'll be happy to merge your PR if you drop the second project :)